weather-app-orig/js/main.js

```diff
@@ -0,0 +1,68 @@
+/* Javascript goes here! */
+
+$(document).ready(function(){
+
+
+  $('.getWeatherData').submit(function(event){
+    event.preventDefault();
+    //we have to prevent the default event so the page doesn't
+    //refresh every time it loads.
+    requestWeatherData();
+
+    //NOTE: So, how is it that you can use the dataType to get the name from the
```

Comment:
On line 29 you are serializing the form. This means that javascript is taking all the inputs in that form and using the 'name' attribute to set the query. This saves us time from getting the value of each input and passing it to the data object. Serialize creates that object for us and sends it as a query string to get tacked on the end of the url.